internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -262,9 +262,9 @@ func ParseEnv(e Env) (sdktypes.Env, error) {
 }
 
 type EnvVar struct {
-	EnvID sdktypes.UUID `gorm:"index;type:uuid"`
-	Name  string
-	Value string // not set if is_secret.
+	EnvID sdktypes.UUID `gorm:"primaryKey;index;type:uuid"`
```

Comment:
Maybe you should add *Env to set the foreign key correctly

Reply:
there is foreign key a couple of lines below. I added combined primary to facilitate model access through gorm